docs/library/stochastic/whitenoise/example-0.py

```diff
@@ -0,0 +1,14 @@
+from signalflow import *
+graph = AudioGraph()
+
+#-------------------------------------------------------------------------------
+# Using white noise to control the pitch of an oscillator.
+# A new pitch is determined once every second. Interpolation is turned off so 
+# that the oscillator jumps to the new pitch instead of smoothly moving to it.
+# Random interval is turned off so that pitch changes occur at a regular rate.
+#-------------------------------------------------------------------------------
+frequency = WhiteNoise(1, 100, 1000, False, False)
```

Comment:
So I wonder if, for some node classes with more arguments, it could be helpful to explicitly write out the arguments using their keywords, so the reader can instantly parse what they do without having to look at the docs. Something like this:

```
frequency = WhiteNoise(frequency=1,
                       min=100,
                       max=1000,
                       interpolate=False,
                       random_interval=False)
```

Thoughts?

Reply:
Yes love this